### PR TITLE
[s] Necropolis tendril destruction is now logged

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
Destroying a ghost spawner is bad, it's a pain, sometimes even impossible, to figure out who did it.

## How This Contributes To The Skyrat Roleplay Experience
Admins knowing things is good.

## Changelog

:cl:
admin: Ashwalker tendril destruction is now logged
/:cl:

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
Cant you just use the logs of attacking it?

**Comment by Zonespace27** [comment]
> Cant you just use the logs of attacking it?

Happened last round, not a single log that could concisely pin it on anyone, no logs relating to that tendril specifically being hit

**Comment by Gandalf2k15** [comment]
> > Cant you just use the logs of attacking it?
> 
> Happened last round, not a single log that could concisely pin it on anyone, no logs relating to that tendril specifically being hit

what if there are two people near it? You should log both. Just encase.

**Comment by Zergspower** [comment]
OH The main tendril not a normal tendril

**Comment by Zonespace27** [comment]
> > > Cant you just use the logs of attacking it?
> > 
> > 
> > Happened last round, not a single log that could concisely pin it on anyone, no logs relating to that tendril specifically being hit
> 
> what if there are two people near it? You should log both. Just encase.

done

File: code/modules/mapfluff/ruins/objects_and_mobs/ash_walker_den.dm [hunk 1]
```diff
@@ -33,6 +33,16 @@
 	ashies = null
 	QDEL_NULL(linked_objective)
 	STOP_PROCESSING(SSprocessing, src)
+	//SKYRAT EDIT START
+	var/compiled_string = "The [src] has been destroyed at [loc_name(src.loc)], nearest mobs are"
```
Comment (Zonespace27) [inline code/modules/mapfluff/ruins/objects_and_mobs/ash_walker_den.dm]:
```suggestion
	var/compiled_string = "The [src] has been destroyed at [loc_name(src.loc)], nearest mobs are "
```

**Pull request merged**